lib/pure/algorithm.nim

```diff
@@ -272,7 +339,6 @@ func sort*[T](a: var openArray[T],
   ## .. code-block:: nim
```

Comment:
I cannot comment above (line 332):

This need some breathing room by separating paragraphs. Add an empty comment line like this:

```
  ## is guaranteed to be stable and the worst case is guaranteed to
  ## be O(n log n).
  ##
  ## The current implementation uses an iterative
  ## mergesort to achieve this. It uses a temporary sequence of
```